app/models/cart.rb

```diff
@@ -26,6 +26,12 @@ def subtotal(item)
     item.price * @contents[item.id.to_s]
   end
 
+  def subtotal_with_discounts(item, quantity)
```

Comment:
This will use the items model method to display the altered subtotal, but if the user checks out right now the price in the item order will not be affected, next step is to alter the controller method